sitewit/services.py

```diff
@@ -35,17 +35,24 @@ def _get_account_auth_header(self, account_token):
         return self._compose_auth_header((
             self._partner_id, self._partner_token, account_token))
 
-    def _get_partner_auth_header(self, subpartner_id=None):
+    def _get_partner_auth_headers(self, subpartner_id=None, remote_id=None):
+        headers = {}
         auth_list = [self._partner_id, self._partner_token]
+
         if subpartner_id is not None:
             auth_list.append(subpartner_id)
-        return self._compose_auth_header(auth_list)
+        elif remote_id is not None:
+            headers['RemoteSubPartnerId'] = base64.b64encode(remote_id)
+
+        headers.update(self._compose_auth_header(auth_list))
+        return headers
 
     def _compose_auth_header(self, elements):
         return {'PartnerAuth': base64.b64encode(':'.join(elements))}
 
     def create_account(self, site_id, url, user_name, user_email,
-                       currency, country_code, user_token=None):
+                       currency, country_code, user_token=None,
+                       remote_id=None):
```

Comment:
But we I think we should name it `remote_subpartner_id` as well, what do you think?